widget/embedded/src/containers/Inputs/Inputs.tsx

```diff
@@ -40,10 +41,14 @@ export function Inputs(props: PropTypes) {
   } = useQuoteStore();
   const { connectedWallets, getBalanceFor } = useWalletsStore();
   const fromTokenBalance = fromToken ? getBalanceFor(fromToken) : null;
-
   const fromTokenFormattedBalance =
     formatBalance(fromTokenBalance)?.amount ?? '0';
 
+  const inputUsdAmount =
+    !inputUsdValue || !fromToken ? new BigNumber(0) : inputUsdValue;
+  const outputUsdAmount =
+    !outputUsdValue || !toToken ? new BigNumber(0) : outputUsdValue;
```

Comment:
I believe it is better to fix this issue in the store, rather than using derived state. We have two functions for resetting the state of the quote store that will be invoked in this situation. You can set `outputUsdValue` to `null` in `resetFromBlockchain`.

Reply:
fixed